packages/babel-plugin-proposal-object-rest-spread/src/shouldStoreRHSInTemporaryVariable.ts

```diff
@@ -1,6 +1,6 @@
 import { types as t } from "@babel/core";
 
-const { isObjectProperty } = t;
+const { isObjectProperty, isArrayPattern, isObjectPattern, isAssignmentPattern, isRestElement, isIdentifier } = t;
```

Comment:
```suggestion
const {
    isObjectProperty,
    isArrayPattern,
    isObjectPattern,
    isAssignmentPattern,
    isRestElement,
    isIdentifier
} = t;
```

Reply:
Changing the deconstruction to this structure might solve the linting error in CI.